packages/material-ui/src/ExpansionPanel/ExpansionPanel.test.js

```diff
@@ -115,6 +115,22 @@ describe('<ExpansionPanel />', () => {
     assert.strictEqual(wrapper.hasClass(classes.disabled), true);
   });
 
+  it('should handle the CollapseComponent prop', () => {
+    const CustomCollapse = props => <div {...props} />;
+
+    const wrapper = shallow(
```

Comment:
We try to avoid `shallow` for new tests. It rarely fails on actual bugs. Oftentimes all it does is catch refactorings.

Reply:
So `mount` right ?

Reply:
@ptbrowne Yes. I know that `mount` is often associated with integration tests but that doesn't translate very well to UI primitives.

Reply:
https://github.com/mui-org/material-ui/pull/14617/commits/9e1937063b00f56ced8b370f3a48adf445eba1b4